src/shared/features/AppletSettings/AppletSettings.styles.ts

```diff
@@ -6,6 +6,7 @@ export const StyledAppletSettingsDescription = styled(Box)`
   max-width: 54.6rem;
   margin-bottom: ${theme.spacing(2.4)};
   color: ${variables.palette.on_surface_variant};
+  font-weight: ${variables.font.weight.bold};
```

Comment:
Do we have some changes in mockups to make it bold?

Reply:
This change also affects subtitles in other applet settings.
For example:
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/e6271206-d773-4a36-9fe8-a199bfe31c69)
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/6daf9854-6857-4e29-b37c-9e3442f60040)

Reply:
Ah, you're right, thank you for catching this. There haven't been updates to those pages yet as far as I'm aware. I will fix